pkg/types/installconfig.go

```diff
@@ -182,3 +186,18 @@ type ClusterNetworkEntry struct {
 	// This is the length in bits - so a 9 here will allocate a /23.
 	DeprecatedHostSubnetLength int32 `json:"hostSubnetLength,omitempty"`
 }
+
+// Proxy defines the proxy settings for the cluster.
```

Comment:
Provide guidelines like 
- atleast one of `HTTPProxy` and `HTTPSProxy` must be set. that matches our struct level validations.